modules/new_serverpod_auth/serverpod_auth_migration/serverpod_auth_migration_client/dartdoc_options.yaml

```
@@ -0,0 +1,5 @@
+dartdoc:
+  categories: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)